tests/network/sriov.go

```diff
@@ -517,6 +519,8 @@ var _ = Describe("[Serial]SRIOV", func() {
 			It("[test_id:1755]should create a virtual machine with two sriov interfaces referring the same resource", func() {
 				sriovNetworks := []string{sriovnet1, sriovnet2}
 				vmi := getSriovVmi(sriovNetworks, defaultCloudInitNetworkData())
+				vmi.Spec.Domain.Devices.Interfaces[1].PciAddress = "0000:81:00.1"
+				vmi.Spec.Domain.Devices.Interfaces[2].PciAddress = "0000:81:00.2"
```

Comment:
Please consider using lower PCI address (0x20 or so) - this will make the domain xml file shorter, and make the guest OS boot a bit faster.

The network interfaces are not always in the expected order so its problematic to hard code the indices.

Reply:
I used the values that would have been used if I did not specified it myself.